blocks/large-promo-block/README.md

```diff
@@ -1,18 +1,25 @@
 # `@wpmedia/large-promo-block`
 
-_Block containing a large promo component. Please provide a 1-2 sentence description of what the block is and what it does._
+The large promo block displays an outline, headline, description, and image.
```

Comment:
```suggestion
The large promo block displays an overline, headline, description, and image.
```

Reply:
Thanks Jack. Fixed it in [f15f237](https://github.com/WPMedia/arc-themes-blocks/pull/1411/commits/f15f2370bcb25fcfd91570ac36b6f4e8f54485d3)